src/main/java/io/gravitee/notifier/email/EmailNotifier.java

```diff
@@ -85,10 +85,10 @@ public CompletableFuture<Void> doSend(final Notification notification, final Map
                     mailMessage,
                     e -> {
                         if (e.succeeded()) {
-                            logger.debug("Email {) has been send successfully! " + e.result().getMessageID());
+                            logger.debug("Email {} has been send successfully!", e.result().getMessageID());
                             future.complete(null);
                         } else {
-                            logger.error("An error occurs while sending email", e.cause());
+                            logger.error("An error occurs while sending email: {}", e.cause());
```

Comment:
not necessary as logger.error take a throwable a second parameter.

Reply:
indeed